.github/workflows/SonarCloud.yml

```diff
@@ -32,5 +32,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        run: ./gradlew build test sonarqube --scan
+        run: ./gradlew build testDebug sonarqube --scan
```

Comment:
What is is the purpose of this change?

Reply:
@Peter-John-paystack I'm putting a `TestActivity` in the debug source folder to use for Robolectric tests

Reply:
@michael-paystack  But test runs debug variant, correct me if im wrong? Have you run `gradle test` locally to see if it doesnt run your `TestActivity`

Reply:
Yea @Peter-John-paystack, I have. It doesn't include the debug source set